starfish/test/full_pipelines/api/test_merfish.py

```diff
@@ -137,113 +130,92 @@ def test_merfish_pipeline_cropped_data():
-          4.88366824e-07, 4.29884250e-08],
-         [1.87091288e-05, 2.46976761e-05, 4.24685823e-05, 6.54237326e-05,
-          7.27664267e-05, 5.22288805e-05, 2.27515571e-05, 5.79549491e-06,
-          8.53010090e-07, 9.15320981e-08]],
+        [[5.60925116e-07, 1.82173255e-06, 3.89044089e-06, 5.88891680e-06,
+          8.33230115e-06, 1.50237584e-05, 2.87189505e-05, 4.38871684e-05,
+          4.99693815e-05, 4.56536509e-05],
+         [8.81411779e-08, 4.10318847e-07, 1.28551540e-06, 3.10769201e-06,
+          7.19512946e-06, 1.69029190e-05, 3.36881749e-05, 4.95603599e-05,
+          5.25721152e-05, 4.34861540e-05],
+         [5.08433935e-08, 2.27155240e-07, 8.00648024e-07, 2.18293940e-06,
+          5.27809016e-06, 1.18301268e-05, 2.17682355e-05, 2.94669844e-05,
+          2.86967188e-05, 2.15504224e-05],
+         [1.97825443e-07, 3.19542323e-07, 7.57550026e-07, 1.62298409e-06,
+          3.05401727e-06, 5.33136472e-06, 8.05869877e-06, 9.40890641e-06,
+          8.13688530e-06, 5.48969501e-06],
+         [1.33133869e-06, 8.94252594e-07, 1.00576995e-06, 1.39044494e-06,
+          1.76213382e-06, 1.97364784e-06, 2.00888189e-06, 1.76678384e-06,
+          1.25888687e-06, 7.36380287e-07],
+         [6.46766375e-06, 3.29912632e-06, 2.29117222e-06, 2.09879431e-06,
+          1.84626458e-06, 1.26157420e-06, 6.53917539e-07, 2.88895023e-07,
+          1.24998053e-07, 5.51619870e-08],
+         [1.83563425e-05, 9.45022111e-06, 6.59115449e-06, 6.00129715e-06,
+          5.01134082e-06, 2.93008884e-06, 1.06252969e-06, 2.31487888e-07,
+          3.21643476e-08, 4.02377687e-09],
+         [2.90175176e-05, 1.75729056e-05, 1.63411660e-05, 1.88506892e-05,
+          1.77172697e-05, 1.07103797e-05, 3.77092488e-06, 7.38105257e-07,
+          7.84077443e-08, 4.51243887e-09],
+         [2.59118187e-05, 2.19022641e-05, 3.13037344e-05, 4.66329184e-05,
+          4.93988118e-05, 3.14469653e-05, 1.12378011e-05, 2.16637454e-06,
+          2.19524679e-07, 1.17655485e-08],
+         [1.41113314e-05, 1.96126693e-05, 4.19564531e-05, 7.57892703e-05,
+          8.83928005e-05, 5.92581382e-05, 2.17709458e-05, 4.22853691e-06,
+          4.23566917e-07, 2.57911594e-08]],
         dtype=np.float32
     )
     assert np.allclose(
         expected_scaled_low_passed,
-        low_passed.numpy_array[5, 0, 0, 40:50, 45:55]
+        scaled_image.numpy_array[5, 0, 0, 40:50, 45:55]
     )
 
-    # detect and decode spots
-    psd = PixelSpotDetector(
-        codebook=experiment.codebook,
-        metric='euclidean',
-        distance_threshold=0.5176,
-        magnitude_threshold=5e-5,
-        min_area=2,
-        max_area=np.inf,
-        norm_order=2,
-        crop_size=(0, 40, 40)
-    )
-    spot_intensities, prop_results = psd.run(low_passed)
+    spot_intensities = merfish.initial_spot_intensities
 
     # verify that the number of spots are correct
     spots_passing_filters = spot_intensities[Features.PASSES_THRESHOLDS].sum()
-    assert spots_passing_filters == 1125
```

Comment:
this is a significant change.  expected?  is this because of the magic scale factors thing?